openpype/plugins/publish/validate_editorial_asset_name.py

```diff
@@ -62,17 +60,21 @@ def process(self, context):
             wrong_names = {}
             self.log.debug(
                 ">> assets_missing_name: {}".format(assets_missing_name))
-            for asset in assets_missing_name.keys():
+
+            for asset in assets_missing_name:
                 _asset = asset.lower().replace("_", "")
-                if _asset in [a.lower().replace("_", "")
-                              for a in asset_db_docs.keys()]:
-                    wrong_names.update({
-                        "required_name": asset,
-                        "used_variants_in_db": [
-                            a for a in asset_db_docs.keys()
-                            if a.lower().replace("_", "") == _asset
-                        ]
-                    })
+                if _asset in [
+                    a.lower().replace("_", "") for a in asset_db_docs
+                ]:
+                    wrong_names.update(
```

Comment:
I'm not sure why are you updating same keys of dictionary on each missing asset. Each not first wrong name will overridde data of previous.